### PR TITLE
REV-164-create-listings-page-stats

### PR DESCRIPTION
**Pull request by IbrahimAldereni-socuimtech**
### Jira URL

https://optimumpartners.atlassian.net/browse/REV-164

### Description

- add save, view, share for listing and user

### Dependencies in Other PRs,Repo

- backend PR: https://github.com/abukhadraop-or/reverifi-api/pull/39

### CODE REVIEW

File: src/components/agent-list/AgentCard/agent-card.styles.js [hunk 1]
```diff
@@ -99,14 +96,12 @@ export const CompanyNamePlaceholder = styled.div`
   margin: 0 0 0.6rem 0;
 `;
 
-export const ButtonsContainer = styled.div`
+export const FooterButtonsContainer = styled.div`
```
Comment (mohdabed123) [inline src/components/agent-list/AgentCard/agent-card.styles.js:99]:
```suggestion
export const ButtonsContainer = styled.div`
```

File: src/components/listing-page/Offer/index.jsx [hunk 2]
```diff
@@ -25,7 +24,7 @@ import React from 'react';
  *
  * @return {JSX.Element}
  */
-function Offer({ price, address }) {
+function Offer({ price, address, id }) {
```
Comment (mohdabed123) [inline src/components/listing-page/Offer/index.jsx:27]: update docs

File: src/components/listing-page/Overview/index.jsx [hunk 3]
```diff
@@ -22,35 +23,44 @@ import {
...
-              // eslint-disable-next-line jsx-a11y/label-has-associated-control
-              <label key={key}>
-                {key}: <Value>{statistics[key]}</Value>
-                {index !== array.length - 1 && <Partition />}
-              </label>
-            ))}
+            <span>
+              Time on reverifi:
+              <Value>
+                &nbsp;
```
Comment (mohdabed123) [inline src/components/listing-page/Overview/index.jsx:35]:
```suggestion
```

File: src/components/shared/SaveAndShareButtons/index.jsx [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  viewOrShareUserOrListing,
+} from 'services/social-statistics';
+import {
+  ButtonsContainer,
+  HeartIcon,
+  RoundedButton,
+  ShareIcon,
+} from './save-and-share-buttons.styles';
+
+function SaveAndShareButtons({ userId, listingId, setShowModal, small }) {
```
Comment (mohdabed123) [inline src/components/shared/SaveAndShareButtons/index.jsx:18]: add docs

File: src/components/shared/SaveAndShareButtons/save-and-share-buttons.styles.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    background-color: ${colors.green};
+    svg path {
+      stroke: ${colors.white};
+    }
+  `}
+`;
+
+export const HeartIcon = styled(Heart)`
+  width: ${({ small }) => (small ? '1rem' : '1.125rem')};
+  height: ${({ small }) => (small ? '1.1rem' : '1.125rem')};
```
Comment (mohdabed123) [inline src/components/shared/SaveAndShareButtons/save-and-share-buttons.styles.js:45]: can't we just use the same size without this , it doesn't really make much of a difference

File: src/components/shared/SaveAndShareButtons/save-and-share-buttons.styles.js [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+  gap: ${({ small }) => (small ? '0.2rem' : '0.5rem')};
+`;
+
+export const RoundedButton = styled(Button)`
+  background-color: ${colors.wildSand};
+  border-radius: 50%;
+  color: ${colors.dustyGray};
+  height: ${({ small }) => (small ? '1.5rem' : '1.87rem')};
+  margin: 0.12rem;
+  padding: ${({ small }) => (small ? '0 0.23rem' : '0 0.37rem')};
```
Comment (mohdabed123) [inline src/components/shared/SaveAndShareButtons/save-and-share-buttons.styles.js:20]: same thing here , not much of a difference

**Review by mohdabed123 — CHANGES_REQUESTED** [review]

File: src/components/shared/SaveAndShareButtons/save-and-share-buttons.styles.js (hunk 6 above)
Comment (IbrahimAldereni-socuimtech) [reply src/components/shared/SaveAndShareButtons/save-and-share-buttons.styles.js:20]: yes i need them like that cuz these are small icons and these slight differences affect the design

**Review by mohdabed123 — APPROVED** [review]

**Pull request merged**